BuildHelpers/Public/Get-ModuleAliases.ps1

```diff
@@ -41,19 +43,19 @@ function Get-ModuleAliases {
         $params = @{
             Force = $True
             Passthru = $True
-            Name = $Name
+            Name = Get-FullPath $Name
```

Comment:
I assume this will always be a path, but comment based help could use a tweak to avoid confusion (i.e. it's not either a name or path, it's a path and only a path)?